service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/OrderIssuedEmailContentProvider.java

```diff
@@ -71,6 +73,34 @@ public OrderIssuedEmailContentProvider(@Value("${ccd.ui.base.url}") String uiBas
             .build();
     }
 
+    public Map<String, Object> buildOrderNotificationParametersForRepresentatives(final CaseDetails caseDetails,
+                                                                              final String localAuthorityCode,
+                                                                              final byte[] documentContents,
+                                                                              final IssuedOrderType issuedOrderType) {
+        CaseData caseData = objectMapper.convertValue(caseDetails.getData(), CaseData.class);
+
+        return ImmutableMap.<String, Object>builder()
+            .put("orderType", getTypeOfOrder(caseData, issuedOrderType))
+            .put("callout", (issuedOrderType != NOTICE_OF_PLACEMENT_ORDER)
+                ? "^" + buildSubjectLineWithHearingBookingDateSuffix(buildSubjectLine(caseData),
+                caseData.getHearingDetails()) : "")
```

Comment:
maybe can be extracted to method, as it is hard to read and it is repeated in another method in this class

Reply:
👍